src/test/java/build/buf/protovalidate/FormatTest.java

```diff
@@ -15,18 +15,107 @@
 package build.buf.protovalidate;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
+import com.google.protobuf.Duration;
 import org.junit.jupiter.api.Test;
+import org.projectnessie.cel.common.types.DoubleT;
+import org.projectnessie.cel.common.types.Err.ErrException;
 import org.projectnessie.cel.common.types.ListT;
+import org.projectnessie.cel.common.types.StringT;
 import org.projectnessie.cel.common.types.UintT;
+import org.projectnessie.cel.common.types.pb.DefaultTypeAdapter;
 import org.projectnessie.cel.common.types.ref.Val;
 
 class FormatTest {
   @Test
-  void largeDecimalValuesAreProperlyFormatted() {
+  void testNotEnoughArgumentsThrows() {
+    StringT one = StringT.stringOf("one");
+    ListT val = (ListT) ListT.newValArrayList(null, new Val[] {one});
+
+    assertThatThrownBy(
+            () -> {
+              Format.format("first value: %s and  %s", val);
+            })
+        .isInstanceOf(ErrException.class)
+        .hasMessageContaining("format: not enough arguments");
+  }
+
+  @Test
+  void testDouble() {
```

Comment:
@jchadwick-buf this is what I meant by some nuance in testing. We could move these double tests to the conformance tests I guess, but not sure about things like testing the above thrown exception. Wdyt?

Reply:
Personally, I'm not overly concerned about handling all of the error cases identically as long as the error cases do fail as-expected, so I don't see this as a huge problem for adding tests to conformance per-se.